Update lib.rs to latest mpl_token_metadata package

dipanshuhappy opened the pull request:
The previous code didn't work for the new mpl token metadata package, this made it confusing compiling in the tutorial for solg beta playground

dipanshuhappy committed:
Update lib.rs to latest mpl_token_metadata package

The previous code didn't work for the new mpl token metadata package

Woody4618 reviewed (changes requested):
Hey, thank you for the PR. You did quite some changes to the code. Added new program id and removed the randomness logic. Did you want this to be merged or just show how to fix it? I have these changes already in a branch, so i would maybe decline the PR, or do you want to do the logic back to how it was before and update thePR?

dipanshuhappy commented:
> Hey, thank you for the PR. You did quite some changes to the code. Added new program id and removed the randomness logic. Did you want this to be merged or just show how to fix it? I have these changes already in a branch, so i would maybe decline the PR, or do you want to do the logic back to how it was before and update thePR? 

If its solved ... awesome you can close this pr ... if you need any other update let me know

Woody4618 commented:
Thank you very much for the PR. I fixed it in a different PR a while back. Yours had a few too many changes. Appreciate it though!
